src/pages/Código bomba.wziwg.js

```diff
@@ -1,7 +1,88 @@
 // API Reference: https://www.wix.com/velo/reference/api-overview/introduction
 // “Hello, World!” Example: https://learn-code.wix.com/en/article/1-hello-world
+import wixStorage from 'wix-storage';
+
+import { utils_config_items, 
+    utils_fmt_only_number,
+    utils_get_elements_values,
+    utils_show_hide_section,
+    SECTION_STATE_LOADING,
+    SECTION_STATE_DATA,
+    SECTION_STATE_NO_DATA,
+} from "public/utils";
+
+import {
+    be_mod_utils_get_bombas_code,
+} from "backend/be_mod_utils";
+
+
+let g_codigo_bomba = [
+    {ui: "#inputCodBomba1", type: "value", format: utils_fmt_only_number, onChange: get_bomba_suggestion},
+    {ui: "#inputCodBomba2", type: "value", format: utils_fmt_only_number, onChange: get_bomba_suggestion},
```

Comment:
don't forget to also change the focus to the next item after onChange
also, when pressing backspace it should delete and return to the previous digit

Reply:
Yes, nice point! For now it's work, but I added on my TODO list.